self-contained/bootstrap_install.sh

```diff
@@ -60,6 +60,9 @@ installContainerRuntime() {
 }
 
 installNetworkPlugin() {
+    if [[ "${NETWORK_PLUGIN}" = "none" ]]; then
+        return
+    fi
```

Comment:
@timmy-wright did you change it intentionally?

Reply:
Yeah, this was going to be part of my next PR. We can either remove it from this one or keep it.